tests/integration/view/simple/simple_test.go

```diff
@@ -160,3 +160,101 @@ func TestView_SimpleWithFieldSubset_ErrorsSelectingExcludedField(t *testing.T) {
 
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestView_SimpleWithExtraFieldInViewSDL(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple view with extra field in SDL",
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type User {
+						name: String
+						age: Int
+					}
+				`,
+			},
+			testUtils.CreateView{
+				Query: `
+					User {
+						name
+					}
+				`,
+				// `age` is present in SDL but not the query
+				SDL: `
+					type UserView {
+						name: String
+						age: Int
+					}
+				`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"name":	"John"
+				}`,
+			},
+			testUtils.Request{
+				Request: `query {
+							UserView {
+								name
+							}
+						}`,
+				Results: []map[string]any{
+					{
+						"name": "John",
+					},
+				},
+			},
+		},
+	}
+
+	testUtils.ExecuteTestCase(t, test)
+}
+
+func TestView_SimpleWithExtraFieldInViewQuery(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple view with extra field in view query",
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type User {
+						name: String
+						age: Int
+					}
+				`,
+			},
+			testUtils.CreateView{
+				// `age` is present in the query but not the SDL
+				Query: `
+					User {
+						name
+						age
+					}
+				`,
+				SDL: `
+					type UserView {
+						name: String
+					}
+				`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"name":	"John"
+				}`,
+			},
+			testUtils.Request{
+				Request: `query {
+							UserView {
+								name
+							}
+						}`,
```

Comment:
nitpick: indentation here (and in above query) is a bit off

Reply:
cheers, will fix

- [ ] Fix indentation